### DISCUSSION TITLE
[BUG]  Why does the LAN deny access

### DISCUSSION THREAD

**Discussion by yuzhongrong** [Chia-Network/chia-blockchain#4602] [Support]
curl --insecure --cert /home/ssl/ipfsnets/.chia/mainnet/config/ssl/wallet/private_wallet.crt --key /home/ssl/ipfsnets/.chia/mainnet/config/ssl/wallet/private_wallet.key -d '{"wallet_id": 1, "new_address":true}' -H "Content-Type: application/json" -X POST https://192.168.3.202:9256/get_next_address

**Comment by yuzhongrong** [comment]
the result is   curl: (7) Failed connect to 192.168.3.202:9256; Connection refused

**Comment by yuzhongrong** [comment]
the node server ip is  192.168.3.202  and the wallet rpc server is 9256 of port

**Comment by yuzhongrong** [comment]
The above request was executed on 192.168.3.200

**Comment by yuzhongrong** [comment]
![image](https://user-images.githubusercontent.com/7382234/117288422-111b4e00-ae9e-11eb-930a-e1b8e955f5dc.png)

**Comment by yuzhongrong** [comment]
ht chia_wallet server is ok

**Comment by yuzhongrong** [comment]
but to request the 9256 from 192.169.3.200 fail

**Comment by Leozolt** [comment]
> but to request the 9256 from 192.169.3.200 fail

Is this a typo or do you try to reach 192.168..3.202 from 192.169.3.200?

If you want to communicate from 192.168.xxx.xxx to 192.169.xxx.xxx you would need to set your network mask to 255.0.0.0

**Comment by olafbuitelaar** [comment]
your wallet seems to listen on 127.0.0.1, instead of 192.168.3.202 or 0.0.0.0. I suspect you need to change the config.yaml to let it listen to either 192.168.3.202 or 0.0.0.0.

**Comment by yuzhongrong** [comment]
> your wallet seems to listen on 127.0.0.1, instead of 192.168.3.202 or 0.0.0.0. I suspect you need to change the config.yaml to let it listen to either 192.168.3.202 or 0.0.0.0.

**Comment by yuzhongrong** [comment]
I saw the configuration file, but I don’t know how to modify the other part.  thanks

**Comment by yuzhongrong** [comment]
Can you elaborate?

**Comment by yuzhongrong** [comment]
![image](https://user-images.githubusercontent.com/7382234/117360421-5a45bf00-aeeb-11eb-81b7-f691604b6a2a.png)

**Reply by sumilying** [reply]
Hello, which items did you change  in the config? Did you solve it? Could you share it?

**Comment by Leozolt** [comment]
full_node_peer sounds like the place where you look (not listen)

I'm also interested in the solution so I hope someone knows where the deamon? listen hostname/ip setting is


might be self_hostname (on 192.168.3.202 in your case)

**Comment by Leozolt** [comment]
Also this is more a discussion or [Help] than a [BUG]

**Comment by yuzhongrong** [comment]
not ok.about this modify. 
![image](https://user-images.githubusercontent.com/7382234/117533511-d50ff680-b01f-11eb-97a5-b46d42239a4e.png)
![image](https://user-images.githubusercontent.com/7382234/117533530-e953f380-b01f-11eb-9320-2eae1b43390b.png)

**Comment by yuzhongrong** [comment]
Still 127.0.0.1

**Comment by Leozolt** [comment]
still an other variable on the server side. I would try changing the self_host variable on a server eg a computer that serves a sevice, and point the full_node_peer to that computer on the client. but I am a noob and can not give you a full featured answer...

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
